### DISCUSSION TITLE
Do we have access to blocked or removed videos from YT?

### DISCUSSION THREAD

**Discussion by EminiEnterprises** [FreeTubeApp/FreeTube#4263] [Q&A]
I am new to this FreeTube and did not know it existed.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
FT can only access the same info as a normal user would when using YT.

that means we cant access removed videos but u are maybe able to access blocked videos.

Like for example a certain videos can only be accessed by a German IP address u can use a VPN to connect to a server in Germany to watch the video

**Comment by EminiEnterprises** [comment]
Okay, thanks.
